resources/3rdparty/include_xerces.cmake

```diff
@@ -16,7 +16,7 @@ if(USE_XERCESC)
 
 
         # find_package(CURL)
-        message (STATUS "Storm (GSPN) - Linking with Xerces-c ${XercesC_VERSION}: ${XercesC_LIBRARIES}")
+        message (STATUS "Storm - Linking with Xerces-c ${XercesC_VERSION}: ${XercesC_LIBRARIES}")
```

Comment:
Why change this?

Reply:
That was the only place where we distinguish on a separate lib. My goal was to make it consistent and use `Storm - ` everywhere.
But I could revert it again if that makes it more clear.

Reply:
Well, only GSPN is linking to it... :-)

Reply:
Let me revert it to keep it clear.